python/lsst/ip/isr/ampOffset.py

```diff
@@ -420,11 +436,59 @@ def getAmpOffsets(self, im, amps, associations, sides):
                 edgeA = ampsEdges[ampId][ampSide]
                 edgeB = ampsEdges[ampNeighbor][(ampSide + 2) % 4]
                 if ampId < ampNeighbor:
-                    interfaceOffset = self.getInterfaceOffset(ampId, ampNeighbor, edgeA, edgeB)
-                    interfaceOffsetLookup[f"{ampId}{ampNeighbor}"] = interfaceOffset
+                    (
+                        interfaceId,
+                        interfaceOffset,
+                        interfaceOffsetOriginal,
+                        ampEdgeGoodFrac,
+                        minFracFail,
+                        maxOffsetFail,
+                    ) = self.getInterfaceOffset(ampsNames[ampId], ampsNames[ampNeighbor], edgeA, edgeB)
+                    interfaceIds.append(interfaceId)
+                    interfaceOffsetOriginals.append(interfaceOffsetOriginal)
+                    ampEdgeGoodFracs.append(ampEdgeGoodFrac)
+                    minFracFails.append(minFracFail)
+                    maxOffsetFails.append(maxOffsetFail)
+                    interfaceOffsetLookup[f"{ampId:02d}:{ampNeighbor:02d}"] = interfaceOffset
                 else:
-                    interfaceOffset = -interfaceOffsetLookup[f"{ampNeighbor}{ampId}"]
+                    interfaceOffset = -interfaceOffsetLookup[f"{ampNeighbor:02d}:{ampId:02d}"]
                 ampsOffsets[ampId] += interfaceWeight * interfaceOffset
+        quartile_summary = np.percentile(interfaceOffsetOriginals, [0, 25, 50, 75, 100])
```

Comment:
I think you might need a check here to make sure numbers were measured or else this is going to crash on an empty input.  (I don't know if it is possible, but I imagine it might be in the limit of one working amp?)